tests/test_applications.py

```diff
@@ -1,73 +1,88 @@
 import os
 from contextlib import asynccontextmanager
-from typing import AsyncIterator, Callable
+from pathlib import Path
+from typing import (
+    Any,
+    AsyncIterator,
+    Callable,
+    Generator,
+)
 
 import anyio
-import httpx
 import pytest
 
 from starlette import status
 from starlette.applications import Starlette
 from starlette.endpoints import HTTPEndpoint
 from starlette.exceptions import HTTPException, WebSocketException
 from starlette.middleware import Middleware
+from starlette.middleware.base import RequestResponseEndpoint
 from starlette.middleware.trustedhost import TrustedHostMiddleware
+from starlette.requests import Request
 from starlette.responses import JSONResponse, PlainTextResponse
 from starlette.routing import Host, Mount, Route, Router, WebSocketRoute
 from starlette.staticfiles import StaticFiles
-from starlette.types import ASGIApp, Receive, Scope, Send
+from starlette.testclient import TestClient
+from starlette.types import (
+    ASGIApp,
+    Receive,
+    Scope,
+    Send,
+)
```

Comment:
Please stop doing the multiline when not needed.
```suggestion
from starlette.types import ASGIApp, Receive, Scope, Send
```

Reply:
@Kludex You might want to add [`skip-magic-trailing-comma`](https://docs.astral.sh/ruff/settings/#format_skip-magic-trailing-comma) to the Ruff formatter config, it would collapse multiline statements that fit on one line such as this one.

Reply:
PR welcome.